synapse/cores/ram.py

```diff
@@ -175,6 +180,24 @@ def _getSizeByProp(self, prop, valu=None, mintime=None, maxtime=None):
 
         return len(rows)
 
+    def _getCoreType(self):
+        return 'ram'
+
+    def _getBlobValu(self, key, default):
+        v = self._blob_store.get(key, default)
+        if v is s_common.novalu:
+            raise s_common.NoSuchName(name=key, mesg='Blob store has no such key present.')
```

Comment:
problem with always returning None as a default is that the caller doesn't know if the key was present or not; which may matter.  Although this almost makes me wonder if a "hasBlobKey" and "delBlobKey" apis are also useful; since "hasBlobKey" takes place of the funky default handling; and delBlobKey allows removal if needed.